store/precomputed_key/wvm/cli.go

```diff
@@ -0,0 +1,99 @@
+package wvm
+
+import (
+	"time"
+
+	wvmtypes "github.com/Layr-Labs/eigenda-proxy/store/precomputed_key/wvm/types"
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	EnabledFlagName  = withFlagPrefix("enabled")
```

Comment:
is there no env var for the wvm account private key?

Reply:
fixed